tool-testsuite/test/org/antlr/v4/test/tool/TestPerformance.java

```diff
@@ -1108,8 +1108,10 @@ private static long sum(long[] array) {
 	}
 
     protected void compileJavaParser(boolean leftRecursive) throws IOException {
-        String grammarFileName = "Java.g4";
-        String sourceName = leftRecursive ? "Java-LR.g4" : "Java.g4";
+        String grammarFileName = leftRecursive ? "JavaLR.g4"    : "Java.g4";
```

Comment:
:bulb: There is no need to have separate `grammarFileName` and `sourceName` variables anymore. You can use `grammarFileName` for both.